tests/users/test_api_register.py

```diff
@@ -0,0 +1,185 @@
+import unittest
+from http import HTTPStatus
+from unittest.mock import patch, Mock
+import requests
+from requests.exceptions import HTTPError
+from flask import json
+from app import messages
+from tests.base_test_case import BaseTestCase
+from app.api.ms_api_utils import post_request, BASE_MS_API_URL
+from app.api.resources.users import UserRegister
+from tests.test_data import user1
+
+
+
+class TestUserRegistrationApi(BaseTestCase):
+
+    @patch("requests.post")
+    def test_api_register_successful(self, mock_register):
+        mock_response = Mock()
+
+        json_response_code = json.dumps(HTTPStatus.CREATED)
+        message = {'message': f'{messages.USER_WAS_CREATED_SUCCESSFULLY}'}
+        status_code = {'status_code': f"{json_response_code}"}
+        expected_tuple = [message, status_code]
+            
+        
+        mock_response.json.return_value = expected_tuple
+        mock_register.return_value = mock_response
+
+        data = user1
```

Comment:
I don't get this

Reply:
I have few users I could use for testing (was used in making dummy data before. So, I'm just importing that data from the test_data.py rather than creating a user again. will be useful for future test as well like the upcoming issues GET users list.